blazor/pager/pager-with-drop-down.md

```diff
@@ -8,7 +8,7 @@ documentation: ug
 ---
 
 
-# Pager Component with Dropdown
+# Pager with Dropdown in Blazor Pager Component
```

Comment:
change to "Pager with Dropdown in Blazor"

Reply:
Changed